include/zephyr/bluetooth/mesh/brg_cfg_cli.h

```diff
@@ -0,0 +1,313 @@
+ *  @param addr_2    Address of the node in the second subnet
+ *  @param rsp       Pointer to a struct storing the received response from the
+ *                   server, or NULL to not wait for a response.
+ *
+ *  @return 0 on success, or (negative) error code on failure.
+ */
+int bt_mesh_brg_cfg_cli_bridging_table_remove(uint16_t net_idx, uint16_t addr, uint16_t net_idx_1,
+					      uint16_t net_idx_2, uint16_t addr_1, uint16_t addr_2,
+					      struct bt_mesh_bridging_table_status *rsp);
+
+/** @brief Sends a Bridged Subnets Get message to the given destination address
+ *  with the given parameters
+ *
+ *  This function sends a Bridged Subnets Get message to the given destination
+ *  address with the given parameters to get the list of subnets that are
+ *  bridged by the node. The function expects a Bridged Subnets List message as
+ *  a response from the destination node.
+ *
+ *  This method can be used asynchronously by setting @p rsp as NULL. This way
+ *  the method will not wait for response and will return immediately after
+ *  sending the command.
+ *
+ *  When @c rsp is set, the user is responsible for providing a buffer for the
+ *  filtered set of N pairs of NetKey Indexes in
+ *  @ref bt_mesh_bridged_subnets_list::list. If a buffer is not provided, the
+ *  bridged subnets won't be copied.
+
+ *  @param net_idx          Network index to encrypt the message with.
+ *  @param addr             Target node address.
+ *  @param filter_net_idx   Filter and NetKey Index used for filtering
+ *  @param start_idx        Start offset to read in units of Bridging Table state entries
+ *  @param rsp              Pointer to a struct storing the received response
+ *                          from the server, or NULL to not wait for a response.
+ *
+ *  @return 0 on success, or (negative) error code on failure.
+ */
+int bt_mesh_brg_cfg_cli_bridged_subnets_get(uint16_t net_idx, uint16_t addr,
+					    struct bt_mesh_filter_netkey filter_net_idx,
+					    uint8_t start_idx,
+					    struct bt_mesh_bridged_subnets_list *rsp);
+
+/** @brief Sends a Bridging Table Get message to the given destination address
+ *  with the given parameters
+ *
+ *  This function sends a Bridging Table Get message to the given destination
+ *  address with the given parameters to get the contents of the Bridging Table.
+ *  The Bridging Table contains the addresses that are authorized to be bridged
+ *  by the node. The function expects a Bridging Table List message as a
+ *  response from the destination node.
+ *
+ *  This method can be used asynchronously by setting @p rsp as NULL. This way
+ *  the method will not wait for response and will return immediately after
+ *  sending the command.
+ *
+ *  When @c rsp is set, the user is responsible for providing a buffer for the
+ *  filtered set of N pairs of NetKey Indexes in
+ *  @ref bt_mesh_bridging_table_list::list. If a buffer is not provided, the
+ *  bridged addresses won't be copied.
```

Comment:
How about specifying this? Considering, API can return potentially huge list (upto 

```suggestion
 *  filtered set of N pairs of NetKey Indexes in
 *  @ref bt_mesh_bridging_table_list::list. If a buffer is not provided, the
 *  bridged addresses won't be copied. If a buffer size is shorter than received list, only
 *  those many entries that fit in the buffer will be copied from the list, and rest will be 
 *  discarded.
```